### PR TITLE
update register and login view

### LINKED ISSUE #245: Login view
we should change to login view to something that the tugraz login and the custom login are equal. I mean that both have to start one button. This leads that tugraz login is more visible, otherwise persons tend automatically to give in the credentials on the input tags

### LINKED ISSUE #246: sign up view layout
fix sign up view layout. the registration form starting with "or" should be placed on the right side underneath the TU Graz sign up button.

Screenshot from 2021-10-11 10-25-56

### PR DESCRIPTION
**Pull request by rekt-hard**
fix: remove redundant closing div tag for register
ui: add accordion for better overview

closes #245 
closes #246 

### Login View:
Screenshot from 2021-10-19 16-54-16
Screenshot from 2021-10-19 16-54-57

### Sign Up View:
Screenshot from 2021-10-19 16-54-38
Screenshot from 2021-10-19 16-55-13

### CODE REVIEW

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login_user.html [hunk 1]
```diff
@@ -45,28 +45,35 @@ <h3 class="ui login header">{{ _('Log in to Repository') }}</h3>
...
-        </form>
-      </div>
+        <div class="ui styled accordion">
+          <div class="title">
+            <i class="user icon"></i>
+            {{ _('Repository Credentials') }}
+          </div>
+          <div class="content">
+            {%- with form = login_user_form %}
+            <div style="padding-bottom: 20px;padding-top: 6px;">
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login_user.html:60]: we could remove the inline css and use a class instead.

**Review by mb-wali — CHANGES_REQUESTED** [review]
looks good.

* could you please add a screenshot - to see how the signup page would look like when the signup option is disabled.
* extract and add the german translation for the new text - `Repository Credentials`

File: invenio_theme_tugraz/translations/de/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -355,25 +351,29 @@ msgid "Log in to Repository"
 msgstr "Anmelden"
 
 #: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login_user.html:38
-#, fuzzy, python-format
-msgid "Sign in with %(title)s"
+#, python-format
+msgid "Log in with %(title)s"
 msgstr "Einloggen mit"
```
Comment (mb-wali) [inline invenio_theme_tugraz/translations/de/LC_MESSAGES/messages.po:356]:
```suggestion
msgstr "Einloggen mit %(title)s"
```

File: invenio_theme_tugraz/translations/de/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -355,25 +351,29 @@ msgid "Log in to Repository"
...
 #: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login_user.html:46
-#: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html:52
+#: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html:51
 msgid "Or"
 msgstr "Oder"
 
-#: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login_user.html:61
+#: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login_user.html:57
+msgid "Log in with repository credentials"
+msgstr "Einloggen mit Repository Daten"
```
Comment (utnapischtim) [inline invenio_theme_tugraz/translations/de/LC_MESSAGES/messages.po:365]: Suggestion: Loggen Sie sich mit ihrem Repository-Account ein

**Review by mb-wali — APPROVED** [review]

**Pull request merged**